blocks/core/ff_module/ff_module-dropdown-button/ff_module-dropdown-button-component/ff_module-dropdown-button-component.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var React = require('react');
+var template = require('./_ff_module-dropdown-button-component.rt.js');
+var activateDropdowns = require('../ff_module-dropdown-button');
+
+module.exports = React.createClass({
+    displayName:'DropdownButton',
+    propTypes: {
+        list: React.PropTypes.array.isRequired,
+        id: React.PropTypes.string.isRequired,
+        text: React.PropTypes.string.isRequired
+    },
+    render: template,
+    componentDidMount: function(){
+        activateDropdowns();
```

Comment:
Yeah, I feel like keeping the behaviour separate will cause less strange interaction between components if you're happy with it that way.